### PR TITLE
adding my profile to LinkFree

### PR DESCRIPTION
**Pull request by nurakmal920207**

### CODE REVIEW

**Review by krshdev — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/nurakmal920207.json
Comment (Pradumnasaraf) [inline public/data/nurakmal920207.json:22]:
```suggestion
      "icon": "graduation-hat",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey! **nurakmal**  there is still a need of some changes, you can also click on commit suggestion for committing it :)
Thank YOu.

**Review by Panquesito7 — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
@nurakmal920207 Your still missing a dash.

File: public/data/nurakmal920207.json
Comment (KendallDoesCoding) [reply public/data/nurakmal920207.json]: @nurakmal920207 Please commit this into the project ASAP

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Please ignore my above comments, now deleted, saw the first commit instead of the latest commit.

**Comment by KendallDoesCoding** [comment]
@Panquesito7 @emmalearnscode LGTM ✅

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, Thank you for your contribution:D

**Comment by nurakmal920207** [comment]
Thank you for all your  helps and comments. This is my first pull request so I am not sure what should I do next. Can I close this pull request now? Is it successfully merged to the main branch? Appreciate helps in this. Thank you.

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
> Thank you for all your helps and comments. This is my first pull request so I am not sure what should I do next. Can I close this pull request now? Is it successfully merged to the main branch? Appreciate helps in this. Thank you.

You wait for a maintainer to merge it, which I will do now. thanks for contributing!

**Comment by Pradumnasaraf** [comment]
> Thank you for all your  helps and comments. This is my first pull request so I am not sure what should I do next. Can I close this pull request now? Is it successfully merged to the main branch? Appreciate helps in this. Thank you.

Hey, you need not to close this pull request as it is not merged yet. You only close PR if there is some issue with the PR or the changes are not required in the project.

**Comment by nurakmal920207** [comment]
> You wait for a maintainer to merge it, which I will do now. thanks for contributing!

Thank you! I can find my file in the main branch now. However, when I search for my name in the Linkfree, I cannot find it. Any idea why?

**Comment by nurakmal920207** [comment]
> Hey, you need not to close this pull request. You only close PR if there is some issue with the PR or the changes are not required in the project.

Thank you for the clarification!

**Comment by KendallDoesCoding** [comment]
> > You wait for a maintainer to merge it, which I will do now. thanks for contributing!
> 
> Thank you! I can find my file in the main branch now. However, when I search for my name in the Linkfree, I cannot find it. Any idea why?

It deploys by release, the last release was 6 days ago, whenever this repo adds a new feature and makes a new release, your name will show up then.

**Comment by emmalearnscode** [comment]
Yes, it'll show up next time a fix or a feature is deployed :)

**Comment by nurakmal920207** [comment]
I see. I understand now. Thank you so much everyone!

**Pull request merged**
